context-and-compliance/03-environments.md

```diff
@@ -0,0 +1,33 @@
+# 3 - Environments
+
+## 3.1 Application has corresponding environments and hosting
+
+### 3.1.1 `testing` environment
+
+#### Context & justification
+
+The `testing` environment, as the name suggests, is an environment for testing. This is where all development/testing is done. Hence, it should only be accessible to engineers and for internal delivery teams. Self-explanatory.
+
+#### Compliance
+
+* [ ] `testing` for internal delivery team; this environment should only be accessible to engineers
```

Comment:
ditto on wording; "should only be accessed by those within the delivery teams"

Reply:
Resolved in https://github.com/madetech/productionisation/pull/58/commits/376e6824e557e5f137c138b5cfadee9b5bba0e24.